content/page/get-involved.md

```diff
@@ -0,0 +1,63 @@
+---
+author: The OBF
+cover:
+  alt: obf_membership
+  image: /wp-content/uploads/2019/02/obf_membership.jpg
+date: "2019-02-04T19:04:53+00:00"
+guid: https://www.open-bio.org/wp/?page_id=2546
+title: Get Involved
+url: /get-involved/
+aliases:
+  - /membership/
+
+---
+# Join the OBF community
+
+The OBF invites anyone who is involved or plans to become involved in promoting open source and/or open science in a biological field to join its community.
+There are many ways you can become part of our growing community, regardless of your career stage and experience with open source and open science.
+
+## Join our online spaces
+
+The OBF community has a number of different online spaces that are open to everyone and can be used to network and discuss anything related to open bioinformatics. You can:
+
+* [Subscribe to our OBF newsletter](https://mailman.open-bio.org/mailman/listinfo/open-bio-l/) (and if you have news to share, you can [also suggest topics](https://github.com/OBF/newsletter/issues).
+* You can [join our community _Slack_](https://join.slack.com/t/open-bio/shared_invite/zt-1pnswao9y-8igcckVxBXhQHCMweHt_NA), where you will find fellow open bioinformatics folks to chat with.
+* We also have an open [_LinkedIn_ group](https://www.linkedin.com/groups/9539620/), where one can get in touch with folks, find interesting job opportunities and more.
+* Last but not least, we also have active presences on social media, on both [BlueSky](https://bsky.app/profile/openbio.bsky.social) and [Mastodon](https://genomic.social/@openbio).
```

Comment:
How about something like this:

```
* We also have active presences on social media, on both [BlueSky](https://bsky.app/profile/openbio.bsky.social) and [Mastodon](https://genomic.social/@openbio).
* Last but not least, our [member projects](/projects) like Biopython have their own communication channels.
```

Sadly that may need to wait and be done as part of #73.

Reply:
Oh, great catch!